onnxruntime/test/framework/distance_test.cc

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/util/distance.h"
+#include <gtest/gtest.h>
+using testing::Types;
+
+namespace onnxruntime {
+
+template <typename T>
+class SqeuclideanTest : public ::testing::Test {
+ public:
+  T func;
+};
+
+using MyTypes = Types<Sqeuclidean<double>, SqeuclideanWithEigen<double> >;
+
+TYPED_TEST_CASE(SqeuclideanTest, MyTypes);
+
+TYPED_TEST(SqeuclideanTest, test1) {
```

Comment:
the test only covers the function added to util. How about the end to end for the op level?